templates/home/home_page.html

```diff
@@ -26,52 +27,148 @@ <h1 class="tna-heading-xl search-hero__heading">Find community-generated history
+                                </div>
+                            </li>
+
+                            <li class="tna-column tna-column--width-1-3 tna-column--width-1-2-medium tna-column--width-1-2-small tna-column--full-tiny tna-!--margin-bottom-l">
+                                <div class="tna-card" data-module="tna-card">
+                                    <div class="tna-card__inner">
+                                        <hgroup class="tna-hgroup-s tna-card__heading">
+                                            <h3 class="tna-hgroup__title">
+                                                <a
+                                                        href="/search/catalogue/?group=community&vis_view=tag"
+                                                        class="tna-card__heading-link"
+                                                        data-card-type="card-group-secondary-nav"
+                                                        data-component-name=""
+                                                        data-link-type="Card text"
+                                                        data-card-title=""
+                                                        data-card-position="2"
+                                                >
+                                                    Related tags
+                                                </a>
+                                            </h3>
+                                        </hgroup>
+
+                                        <a
+                                                href="/search/catalogue/?group=community&vis_view=tag"
+                                                class="tna-card__image-container"
+                                                tabindex="-1"
+                                                title=""
+                                                data-card-type="card-group-secondary-nav"
+                                                data-component-name=""
+                                                data-link-type="Card image"
+                                                data-card-title=""
+                                                data-card-position="2"
+                                        >
+                                            <picture class="tna-card__image">
+                                                <img src="/static/images/ohos-homepage-tags.jpg" alt=""  />
+                                            </picture>
+                                        </a>
+
+                                        <div class="tna-card__body">
+                                            <p>Use AI-generated tags to uncover connections to people, places and organisations of the past.</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </li>
```

Comment:
Same as above 2 points for structure, links

Reply:
I've run locally and changes seem good. The list items (cards) remain within the <ul> structure, so I think we are ok.